packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -223,7 +228,7 @@ class Frame extends React.Component {
             </li>
           ) : null}
         <li className={cx('search-wrapper')}>
-          <input {...inputAttrs} value={searchValue} />
+          {(isSearchDisabled) ? <div className={cx('placeholder')}>{placeholder || '\xa0'}</div> : <input {...inputAttrs} value={searchValue} /> }
```

Comment:
`placeholder` is _always_ going to be present ([either default or custom](https://github.com/cerner/terra-core/blob/1061a1a9d6e60ab314a451a4465d30fdfa57020f/packages/terra-form-select/src/MultiSelect.jsx#L199-L200)). What is the need for this condition? ⬇️

```
placeholder || '\xa0'
```

Reply:
I missed that...! this condition is not required we can remove this from other files too.